staging/src/k8s.io/kubectl/pkg/cmd/attach/attach.go

```diff
@@ -110,7 +111,7 @@ func NewCmdAttach(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra
 		},
 	}
 	cmdutil.AddPodRunningTimeoutFlag(cmd, defaultPodAttachTimeout)
-	cmd.Flags().StringVarP(&o.ContainerName, "container", "c", o.ContainerName, "Container name. If omitted, the first container in the pod will be chosen")
+	cmd.Flags().StringVarP(&o.ContainerName, "container", "c", o.ContainerName, "Container name. If omitted, prefer to use the kubectl.kubernetes.io/default-container annotation or the first container in the pod will be chosen")
```

Comment:
I believe you can remove "prefer to ".  I would do:

"If omitted, use the kubectl.kubernetes.io/default-container annotation for selecting the container to be attached or the first container in the pod will be chosen"

/cc @eddiezane WDYT

Reply:
@mengjiao-liu the message for “kubectl cp” should be changed as well

Reply:
Thanks, I have modified it according to the suggestion you gave